mappings/net/minecraft/client/render/entity/tile/SignModel.mapping

```diff
@@ -1 +1,4 @@
 CLASS net/minecraft/class_546 net/minecraft/client/render/entity/tile/SignModel
+	FIELD field_2418 plate Lnet/minecraft/class_163;
```

Comment:
no one calls that bit of the sign a plate

Reply:
then what hould it be csllaed

Reply:
the board

Reply:
obviously

Reply:
a plate would make sense if it were metal I think metal ones are in fact called plates
but mc signs are wooden

Reply:
Fix this